src/components/CorporateCampaign/CampaignHeader.vue

```diff
@@ -1,36 +1,58 @@
 <template>
-	<section class="campaign-header section row align-center">
-		<!-- TODO: add optional Hero Image Presentation -->
-		<!-- eslint-disable-next-line max-len -->
-		<div class="small-10 large-5 xlarge-6 small-order-1 large-order-2 align-self-middle columns campaign-header__cta_wrapper">
-			<div v-if="headerLogo.url" class="campaign-header__logo">
-				<img :title="headerLogo.title" :src="headerLogo.url">
+	<section
+		class="campaign-header section"
+	>
+		<div class="row align-center">
+			<div class="small-12 medium-10 large-6 xlarge-5 small-order-2 large-order-1 columns">
+				<!-- <featured-loans-carousel
+					:disable-redirects="true"
+					:show-view-loan-cta="false"
+					@add-to-basket="handleAddToBasket"
+				/> -->
```

Comment:
we'll bring this in once we create a LoanCardController version of the NoClickLoanCard.